documentation/markdown/usage/account/json-api.md

```diff
@@ -0,0 +1,319 @@
+# Account management JSON API
+
+Everything related to account management is done through a JSON API,
+of which we will describe all paths below.
+There are also HTML pages available to handle account management
+that use these APIs internally.
+Links to these can be found in the HTML controls
+All APIs expect JSON as input, and will return JSON objects as output.
+
+## Finding API URLs
+
+All URLs below are relative to the index account API URL, which by default is `http://localhost:3000/.account/`.
+Every response of an API request will contain a `controls` object,
+containing all the URLs of the other API endpoints.
+It is generally advised to make use of these controls instead of hardcoding the URLs.
+Only the initial index URL needs to be known then to find the controls.
+Certain controls will be missing if those features are disabled in the configuration.
+
+## API requests
+
+Many APIs require a POST request to perform an action.
+When doing a GET request on these APIs they will return an object describing what input is expected for the POST.
+
+## Authorization
+
+After logging in, the API will return a `set-cookie` header of the format `css-account=$VALUE`
+This cookie is necessary to have access to many of the APIs.
+When including this cookie, the controls object will also be extended with new URLs that are now accessible.
+When logging in, the response body JSON body will also contain an `authorization` field
+containing the `$VALUE` value mentioned above.
+Instead of using cookies,
+this value can be used in an `Authorization` header with value `CSS-Account-Cookie $VALUE`
```

Comment:
`CSS-Account-Cookie` might not be an appropriate scheme name, or at least a confusing one. Suggestion: `CSS-API-Token`.

Reply:
Going to change it to `CSS-Account-Token`. `CSS-API-Token` feels too generic just in case another API gets added at some point.